extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -50,7 +50,32 @@
         "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/label": {
           "title": "Tracker URL Label",
           "type": "string",
-          "description": "Human-Friendly label"
+          "description": "Human-Friendly label."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/offerID": {
```

Comment:
Should we create a subsection for Offer related fields in messageInteraction object.
Also, we should update messagefeedback schema also for Offer related fields.

Reply:
ok, did the changes to separate out offers section.
messagefeedback schema changes need to be taken up separately by the right owner (efs team?) as a separate PR.

Reply:
@mjasrotia, yes, this is good to go. Thank you!